{"asctime": "2025-09-27 05:07:37,489", "name": "explorer", "levelname": "INFO", "message": "Attempting to connect to MongoDB at: mongodb://***:***@localhost:27017/"}
{"asctime": "2025-09-27 05:07:37,635", "name": "explorer", "levelname": "INFO", "message": "TTL index created on 'created_at' field for cache expiration after 3600 seconds."}
{"asctime": "2025-09-27 05:07:37,636", "name": "explorer", "levelname": "INFO", "message": "Successfully connected to MongoDB database 'wikidata_explorer'."}
{"asctime": "2025-09-27 05:07:50,847", "name": "explorer", "levelname": "INFO", "message": "Attempting to connect to MongoDB at: mongodb://***:***@localhost:27017/"}
{"asctime": "2025-09-27 05:07:50,872", "name": "explorer", "levelname": "INFO", "message": "TTL index created on 'created_at' field for cache expiration after 3600 seconds."}
{"asctime": "2025-09-27 05:07:50,873", "name": "explorer", "levelname": "INFO", "message": "Successfully connected to MongoDB database 'wikidata_explorer'."}
{"asctime": "2025-09-27 05:10:06,266", "name": "explorer", "levelname": "INFO", "message": "Attempting to connect to MongoDB at: mongodb://***:***@localhost:27017/"}
{"asctime": "2025-09-27 05:10:06,291", "name": "explorer", "levelname": "INFO", "message": "TTL index created on 'created_at' field for cache expiration after 3600 seconds."}
{"asctime": "2025-09-27 05:10:06,292", "name": "explorer", "levelname": "INFO", "message": "Successfully connected to MongoDB database 'wikidata_explorer'."}
{"asctime": "2025-09-27 05:11:26,584", "name": "explorer", "levelname": "INFO", "message": "Attempting to connect to MongoDB at: mongodb://***:***@localhost:27017/"}
{"asctime": "2025-09-27 05:11:26,610", "name": "explorer", "levelname": "INFO", "message": "TTL index created on 'created_at' field for cache expiration after 3600 seconds."}
{"asctime": "2025-09-27 05:11:26,610", "name": "explorer", "levelname": "INFO", "message": "Successfully connected to MongoDB database 'wikidata_explorer'."}
{"asctime": "2025-09-27 05:11:37,795", "name": "explorer", "levelname": "DEBUG", "message": "Cache MISS for hash: 4be11f69a0c285f510c075a37b2ae56e82e5ef8179866f50c4dd08ef142bba8e"}
{"asctime": "2025-09-27 05:11:37,796", "name": "explorer", "levelname": "INFO", "message": "Executing new SPARQL query for hash: 4be11f69a0c285f510c075a37b2ae56e82e5ef8179866f50c4dd08ef142bba8e"}
{"asctime": "2025-09-27 05:11:49,666", "name": "explorer", "levelname": "DEBUG", "message": "Cache entry saved/updated for hash: 4be11f69a0c285f510c075a37b2ae56e82e5ef8179866f50c4dd08ef142bba8e"}
{"asctime": "2025-09-27 05:11:49,668", "name": "explorer", "levelname": "INFO", "message": "New result cached for hash: 4be11f69a0c285f510c075a37b2ae56e82e5ef8179866f50c4dd08ef142bba8e"}
{"asctime": "2025-09-27 05:15:05,786", "name": "explorer", "levelname": "INFO", "message": "Attempting to connect to MongoDB at: mongodb://***:***@localhost:27017/"}
{"asctime": "2025-09-27 05:15:05,811", "name": "explorer", "levelname": "INFO", "message": "TTL index created on 'created_at' field for cache expiration after 3600 seconds."}
{"asctime": "2025-09-27 05:15:05,812", "name": "explorer", "levelname": "INFO", "message": "Successfully connected to MongoDB database 'wikidata_explorer'."}
{"asctime": "2025-09-27 05:16:02,394", "name": "explorer", "levelname": "INFO", "message": "Attempting to connect to MongoDB at: mongodb://***:***@localhost:27017/"}
{"asctime": "2025-09-27 05:16:02,421", "name": "explorer", "levelname": "INFO", "message": "TTL index created on 'created_at' field for cache expiration after 3600 seconds."}
{"asctime": "2025-09-27 05:16:02,422", "name": "explorer", "levelname": "INFO", "message": "Successfully connected to MongoDB database 'wikidata_explorer'."}
{"asctime": "2025-09-27 05:16:16,630", "name": "explorer", "levelname": "INFO", "message": "Attempting to connect to MongoDB at: mongodb://***:***@localhost:27017/"}
{"asctime": "2025-09-27 05:16:16,656", "name": "explorer", "levelname": "INFO", "message": "TTL index created on 'created_at' field for cache expiration after 3600 seconds."}
{"asctime": "2025-09-27 05:16:16,657", "name": "explorer", "levelname": "INFO", "message": "Successfully connected to MongoDB database 'wikidata_explorer'."}
{"asctime": "2025-09-27 05:20:40,149", "name": "explorer", "levelname": "INFO", "message": "Attempting to connect to MongoDB at: ***://localhost:27017/"}
{"asctime": "2025-09-27 05:20:40,174", "name": "explorer", "levelname": "ERROR", "message": "MongoDB connection failed (PyMongo Error): Index already exists with a different name: created_at_ttl_index, full error: {'ok': 0.0, 'errmsg': 'Index already exists with a different name: created_at_ttl_index', 'code': 85, 'codeName': 'IndexOptionsConflict'}"}
{"asctime": "2025-09-27 05:20:49,433", "name": "explorer.views", "levelname": "ERROR", "message": "Error listing queries: Cannot resolve keyword 'query_content' into field. Choices are: created_at, id, name, sparql_query"}
{"asctime": "2025-09-27 05:20:53,482", "name": "explorer.views", "levelname": "INFO", "message": "Cache MISS. Querying Wikidata for query hash: 825f3e4ae0b36d991661696c2a2088004d3a292a9b01075bf823a77b4f10ef88"}
{"asctime": "2025-09-27 05:21:08,949", "name": "explorer.views", "levelname": "ERROR", "message": "Request Error: HTTPSConnectionPool(host='query.wikidata.org', port=443): Read timed out. (read timeout=15)"}
{"asctime": "2025-09-27 05:21:13,294", "name": "explorer.views", "levelname": "INFO", "message": "Cache MISS. Querying Wikidata for query hash: 825f3e4ae0b36d991661696c2a2088004d3a292a9b01075bf823a77b4f10ef88"}
{"asctime": "2025-09-27 05:21:28,410", "name": "explorer.views", "levelname": "ERROR", "message": "Request Error: HTTPSConnectionPool(host='query.wikidata.org', port=443): Read timed out. (read timeout=15)"}
{"asctime": "2025-09-27 05:22:07,978", "name": "explorer", "levelname": "INFO", "message": "Attempting to connect to MongoDB at: ***://localhost:27017/"}
{"asctime": "2025-09-27 05:22:08,015", "name": "explorer", "levelname": "ERROR", "message": "MongoDB connection failed (PyMongo Error): Index already exists with a different name: created_at_ttl_index, full error: {'ok': 0.0, 'errmsg': 'Index already exists with a different name: created_at_ttl_index', 'code': 85, 'codeName': 'IndexOptionsConflict'}"}
{"asctime": "2025-09-27 05:22:16,497", "name": "explorer", "levelname": "INFO", "message": "Attempting to connect to MongoDB at: ***://localhost:27017/"}
{"asctime": "2025-09-27 05:22:16,531", "name": "explorer", "levelname": "ERROR", "message": "MongoDB connection failed (PyMongo Error): Index already exists with a different name: created_at_ttl_index, full error: {'ok': 0.0, 'errmsg': 'Index already exists with a different name: created_at_ttl_index', 'code': 85, 'codeName': 'IndexOptionsConflict'}"}
{"asctime": "2025-09-27 05:22:37,226", "name": "explorer", "levelname": "INFO", "message": "Attempting to connect to MongoDB at: ***://localhost:27017/"}
{"asctime": "2025-09-27 05:22:37,250", "name": "explorer", "levelname": "ERROR", "message": "MongoDB connection failed (PyMongo Error): Index already exists with a different name: created_at_ttl_index, full error: {'ok': 0.0, 'errmsg': 'Index already exists with a different name: created_at_ttl_index', 'code': 85, 'codeName': 'IndexOptionsConflict'}"}
{"asctime": "2025-09-27 05:22:44,129", "name": "explorer.views", "levelname": "ERROR", "message": "Error listing queries: Cannot resolve keyword 'query_content' into field. Choices are: created_at, id, name, sparql_query"}
{"asctime": "2025-09-27 05:22:45,442", "name": "explorer.views", "levelname": "INFO", "message": "Cache MISS. Querying Wikidata for query hash: 825f3e4ae0b36d991661696c2a2088004d3a292a9b01075bf823a77b4f10ef88"}
{"asctime": "2025-09-27 05:22:59,588", "name": "explorer.views", "levelname": "ERROR", "message": "Wikidata HTTP Error: 429 Client Error: Too Many Requests for url: https://query.wikidata.org/sparql?query=%23+Find+all+humans+born+in+1980%0ASELECT+%3Fhuman+%3FhumanLabel+%3FdateOfBirth%0AWHERE+%7B%0A++%3Fhuman+wdt%3AP31+wd%3AQ5%3B++++++++%23+Is+a+human%0A+++++++++wdt%3AP569+%3FdateOfBirth+.+%23+Has+date+of+birth%0A++FILTER+%28YEAR%28%3FdateOfBirth%29+%3D+1980%29%0A++SERVICE+wikibase%3Alabel+%7B+bd%3AserviceParam+wikibase%3Alanguage+%22%5BAUTO_LANGUAGE%5D%2Cen%22.+%7D%0A%7D%0ALIMIT+10"}
{"asctime": "2025-09-27 05:24:02,344", "name": "explorer", "levelname": "INFO", "message": "Attempting to connect to MongoDB at: ***://localhost:27017/"}
{"asctime": "2025-09-27 05:24:02,372", "name": "explorer", "levelname": "ERROR", "message": "MongoDB connection failed (PyMongo Error): Index already exists with a different name: created_at_ttl_index, full error: {'ok': 0.0, 'errmsg': 'Index already exists with a different name: created_at_ttl_index', 'code': 85, 'codeName': 'IndexOptionsConflict'}"}
{"asctime": "2025-09-27 05:25:20,926", "name": "explorer", "levelname": "INFO", "message": "Attempting to connect to MongoDB at: ***://localhost:27017/"}
{"asctime": "2025-09-27 05:25:20,950", "name": "explorer", "levelname": "ERROR", "message": "MongoDB connection failed (PyMongo Error): Index already exists with a different name: created_at_ttl_index, full error: {'ok': 0.0, 'errmsg': 'Index already exists with a different name: created_at_ttl_index', 'code': 85, 'codeName': 'IndexOptionsConflict'}"}
{"asctime": "2025-09-27 05:31:31,600", "name": "explorer", "levelname": "INFO", "message": "Successfully connected to MongoDB database 'wikidata_explorer_new'."}
{"asctime": "2025-09-27 05:31:31,601", "name": "explorer", "levelname": "ERROR", "message": "An unexpected error occurred during MongoDB connection: Collection objects do not implement truth value testing or bool(). Please compare with None instead: collection is not None"}
{"asctime": "2025-09-27 05:31:35,728", "name": "explorer", "levelname": "INFO", "message": "Successfully connected to MongoDB database 'wikidata_explorer_new'."}
{"asctime": "2025-09-27 05:31:35,729", "name": "explorer", "levelname": "ERROR", "message": "An unexpected error occurred during MongoDB connection: Collection objects do not implement truth value testing or bool(). Please compare with None instead: collection is not None"}
{"asctime": "2025-09-27 05:31:42,321", "name": "explorer", "levelname": "INFO", "message": "Successfully connected to MongoDB database 'wikidata_explorer_new'."}
{"asctime": "2025-09-27 05:31:42,322", "name": "explorer", "levelname": "ERROR", "message": "An unexpected error occurred during MongoDB connection: Collection objects do not implement truth value testing or bool(). Please compare with None instead: collection is not None"}
{"asctime": "2025-09-27 05:31:48,083", "name": "explorer.views", "levelname": "ERROR", "message": "Error listing queries: Cannot resolve keyword 'query_content' into field. Choices are: created_at, id, name, sparql_query"}
{"asctime": "2025-09-27 05:48:36,254", "name": "explorer", "levelname": "INFO", "message": "Successfully connected to MongoDB database 'wikidata_explorer_new'."}
{"asctime": "2025-09-27 05:48:36,256", "name": "explorer", "levelname": "ERROR", "message": "An unexpected error occurred during MongoDB connection: Collection objects do not implement truth value testing or bool(). Please compare with None instead: collection is not None"}
